### PR TITLE
Add reference docs for remaining pkg and live commands

### PR DESCRIPTION
**Pull request by mortent**
Updates the reference doc with the remaining commands in the `pkg` and `live` command groups.

Fixes: https://github.com/GoogleContainerTools/kpt/issues/1939

### CODE REVIEW

File: site/reference/live/migrate/README.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+linkTitle: "migrate"
+type: docs
+description: >
+Migrate a package and the inventory object to use the ResourceGroup CRD.
+---
+<!--mdtogo:Short
+    Migrate a package and the inventory object to use the ResourceGroup CRD.
+-->
+
+`migrate` moves the inventory list in the cluster from a ConfigMap resource to a
```
Comment (frankfarzan) [inline site/reference/live/migrate/README.md:12]: `ConfigMap`

go/kpt-docs-style-guide D6
There are other instances

File: site/reference/live/migrate/README.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+description: >
+Migrate a package and the inventory object to use the ResourceGroup CRD.
+---
+<!--mdtogo:Short
+    Migrate a package and the inventory object to use the ResourceGroup CRD.
+-->
+
+`migrate` moves the inventory list in the cluster from a ConfigMap resource to a 
+ResourceGroup CR and moves the inventory information on the local filesystem from
+the inventory template into the Kptfile.
```
Comment (frankfarzan) [inline site/reference/live/migrate/README.md:14]: may want to give a back story that previous versions of kpt used CM

File: site/reference/pkg/cat/README.md [hunk 3]
```diff
@@ -10,77 +10,52 @@ description: >
     Print the resources in a file/directory
 -->
 
-Prints the resources in a file/directory as yaml to stdout.
-
-`cat` is useful for printing the resources in a file/directory which might
-contain other non-resource files.
+`cat` prints the resources in a file or directory as yaml to stdout.
```
Comment (frankfarzan) [inline site/reference/pkg/cat/README.md:13]: @phanimarupaka where did we land on with `tree` vs ` cat` vs `view`? Also, more generally with all commands being verbs.

File: site/reference/live/migrate/README.md (hunk 2 above)
Comment (mortent) [reply site/reference/live/migrate/README.md]: Added a short section with some more background. I assume we will publish a migration guide for kpt v1 and that should probably include some more in-depth explanation of migration. I don't think we want to push too much of that information into the reference docs.

File: site/reference/pkg/cat/README.md (hunk 3 above)
Comment (phanimarupaka) [reply site/reference/pkg/cat/README.md:13]: I feel we should retain `tree` name and deprecate `cat` for now. We din't see strong use-cases for `cat`. We can always add it later if we want to. WDYT?

Comment (phanimarupaka) [reply site/reference/pkg/cat/README.md:13]: `status` as in live status ? I feel we should retain that too as it is highly intuitive and common word.

**Review by frankfarzan — APPROVED** [review]

**Pull request merged**